examples/opengl2_example/imgui_impl_glfw_gl2.cpp

```diff
@@ -129,12 +136,30 @@ void ImGui_ImplGlfwGL2_RenderDrawData(ImDrawData* draw_data)
     glDisableClientState(GL_COLOR_ARRAY);
     glDisableClientState(GL_TEXTURE_COORD_ARRAY);
     glDisableClientState(GL_VERTEX_ARRAY);
+#ifdef __EMSCRIPTEN__
+    if(last_texture)
```

Comment:
@ocornut pay attention to this too. Without this check I have problem to draw custom scene.